demo/src/main/java/com/example/demo/service/WebshopService.java

```diff
@@ -0,0 +1,48 @@
+package com.example.demo.service;
+
+import com.example.demo.exception.ItemNotFoundException;
+import com.example.demo.model.ShopItem;
+import com.example.demo.repository.WebshopRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+@Service
+@RequiredArgsConstructor
+public class WebshopService {
+
+    private final WebshopRepository webshopRepository;
+
+    public List<ShopItem> findByTwoNames(String name1, String name2){
+        final var items = webshopRepository.findByNameOrName(name1, name2);
+        if(items.isEmpty()){
+            throw new ItemNotFoundException("No item is found with name " + name1 + " and name " + name2);
```

Comment:
`and` or `or`? :)

Reply:
@deb05121 odd, because I still do not see the change
![image](https://github.com/deb05121/SpringBootProjects/assets/42215450/82ac6a43-7085-4c6a-9337-eb65b6ad0ca7)